crates/biome_js_formatter/src/ts/module/import_type_assertion.rs

```diff
@@ -0,0 +1,33 @@
+use crate::prelude::*;
+
+use biome_formatter::write;
+use biome_js_syntax::TsImportTypeAssertion;
+use biome_js_syntax::TsImportTypeAssertionFields;
+#[derive(Debug, Clone, Default)]
+pub(crate) struct FormatTsImportTypeAssertion;
+impl FormatNodeRule<TsImportTypeAssertion> for FormatTsImportTypeAssertion {
+    fn fmt_fields(&self, node: &TsImportTypeAssertion, f: &mut JsFormatter) -> FormatResult<()> {
+        let TsImportTypeAssertionFields {
+            assertion_kind,
+            colon_token,
+            l_curly_token,
+            assertions,
+            r_curly_token,
+        } = node.as_fields();
+
+        write!(
+            f,
+            [
+                space(),
```

Comment:
I think this wrong, it should not be here

Reply:
ok, i will remove here.